brewtils/rest/client.py

```diff
@@ -224,7 +225,7 @@ def can_connect(self, **kwargs):
         """Determine if a connection to the Beer-garden server is possible
 
         Args:
-            kwargs: Keyword arguments to pass to Requests session call
+            **kwargs: Keyword arguments to pass to Requests session call
```

Comment:
We were pretty inconsistent about whether we listed it as `kwargs` or ``**kwargs``.  I standardized it to ``**kwargs`` across the board, which seemed like the standard according to the [sphinx examples](https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html#example-google).